rules/windows/powershell/powershell_script/posh_ps_malicious_commandlets.yml

```diff
@@ -167,16 +172,19 @@ detection:
             - 'Invoke-SharpPrintNightmare'
             - 'Invoke-SharpPrinter'
             - 'Invoke-SharpRDP'
+            - 'Invoke-SharpSCCM'
             - 'Invoke-SharpSSDP'
             - 'Invoke-SharpSecDump'
             - 'Invoke-SharpSniper'
             - 'Invoke-SharpSploit'
             - 'Invoke-SharpSpray'
             - 'Invoke-SharpStay'
             - 'Invoke-SharpUp'
+            - 'Invoke-SharpWSUS'
             - 'Invoke-SharpWatson'
             - 'Invoke-Sharphound2'
             - 'Invoke-Sharphound3'
+            - 'Invoke-SharpHound4'
```

Comment:
`Invoke-Sharp` shoud be  enought to cover sharp varian

Reply:
This could be risky as Invoke-Sharp is used a lot. I think it's better to keep it specific. Don't you think?

Reply:
I don't think a legitimate `Invoke-Sharp` cmdlet  exist.
Can make a first cleanup of overlaps contains.
`Invoke-SharpHound` # cover Invoke-SharpHound2, Invoke-SharpHound3,..

Reply:
Yea that should be fine.